mobile/src/main/java/org/hawkular/client/android/fragment/TriggerDetailFragment.java

```diff
@@ -16,33 +16,90 @@
 import org.hawkular.client.android.R;
 import org.hawkular.client.android.backend.model.Trigger;
 import org.hawkular.client.android.util.Fragments;
+import org.jboss.aerogear.android.store.DataManager;
+import org.jboss.aerogear.android.store.generator.IdGenerator;
+import org.jboss.aerogear.android.store.sql.SQLStore;
+import org.jboss.aerogear.android.store.sql.SQLStoreConfiguration;
 
 import android.app.Fragment;
+import android.content.Context;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ImageButton;
+import android.widget.TextView;
 import android.widget.Toast;
+
+import java.util.UUID;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 import icepick.State;
 
 
 public class TriggerDetailFragment extends android.support.v4.app.Fragment {
             @State
             @Nullable
             Trigger trigger;
+    @BindView(R.id.add_button)
+    ImageButton button;
+
+    @BindView(R.id.text_title)
+    TextView textView;
+
 
     @Nullable @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_container,container,false);
+        return inflater.inflate(R.layout.fragment_trigger_detail,container,false);
+    }
+
+    @OnClick(R.id.add_button)
+    public void addTrigger(){
+        Context context = getActivity();
+        final Animation myAnim = AnimationUtils.loadAnimation(context, R.anim.bounce);
+        MyBounceInterpolator interpolator = new MyBounceInterpolator(0.2, 20);
+        myAnim.setInterpolator(interpolator);
+
+        button.setColorFilter(getResources().getColor(R.color.background_primary));
+        button.startAnimation(myAnim);
+
+        textView.setText("Trigger Added");
+        SQLStore<Trigger> store = openStore(context);
+        store.openSync();
+        store.save(trigger);
+    }
+
+    private SQLStore<Trigger> openStore(Context context) {
+        DataManager.config("FavouriteTriggers", SQLStoreConfiguration.class)
+                .withContext(context)
+                .withIdGenerator(new IdGenerator() {
+                    @Override
+                    public String generate() {
+                        return UUID.randomUUID().toString();
+                    }
+                }).store(Trigger.class);
+        return (SQLStore<Trigger>) DataManager.getStore("FavouriteTriggers");
+    }
+
+    private void setUpBindings() {
+        ButterKnife.bind(this, getView());
     }
 
     @Override public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
+
+        setUpBindings();
```

Comment:
I wanna start to kill this kind of unecessarly methods like I did here: https://github.com/hawkular/hawkular-android-client/pull/129

So... I'll like suggest you to kill this method and move the content to `onCreateView`

```java
public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
    View view =  inflater.inflate(R.layout.fragment_trigger_detail, container, false);
    ButterKnife.bind(this, view);
    return view;
}
```